src/main/java/uk/gov/ons/ssdc/caseprocessor/rasrm/model/dto/RasRmPartyAssociationDTO.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.ons.ssdc.caseprocessor.rasrm.model.dto;
+
+import lombok.Data;
+
+@Data
+public class RasRmPartyAssociationDTO {
+  private String businessRespondentStatus; // TODO: could be an enum, but we only check one status
```

Comment:
enum would still be nicer than String matching

Reply:
enum will make the interface brittle though... whenever RAS-RM change _their_ enum, we would have to add new values, or else it will break our code, which is not desirable